### PR TITLE
Update connect buttons in banner

### PR DESCRIPTION
**Pull request by johnHackworth**
#### Changes proposed in this Pull Request:

* This PR updates the looks of the connect buttons to make them jetpack-green.
* It also adds a parameter to the connection URL in the banner buttons, since we are already telling the user that by clicking those button they agree with our ToS, so we don't need to ask again wpcom-side
image

#### Testing instructions:
* Disconnect any of your jetpack sites, and then navigate anywhere where the banner up there is showing. The buttons should be green, and the url they link to should contain "already_approved=true"

ping @joanrho and @MichaelArestad  for design review

### CODE REVIEW

**Comment by joanrho** [comment]
Thanks @johnHackworth, this looks great! I see the changes in the small banner on both wp-admin Dashboard and in the Plugins directory after the full-screen banner there is dismissed.

Some comments:
1. Can you please add a space in between "Set" and "up" in the green button? So the text will read "Set up Jetpack".
2. The button URL doesn’t say “already_approved=true” like you describe, it says “already_authorized=true”. Same thing?

**Comment by MichaelArestad** [comment]
Visually, looks great! Nice work!

**Comment by joanrho** [comment]
I also added another small change to the colors variables and connect button colors.

other than that, LGTM!

**Comment by johnHackworth** [comment]
Joan:
> it says “already_authorized=true”. Same thing?

yup, sorry, I messed up the text in the PR, that's the actual param :)

**Comment by johnHackworth** [comment]
I've just added the full-screen banner button I was still missing to the PR:

image

**Comment by joanrho** [comment]
Just added some copy changes to the full-screen banner (as described in my kickoff post), updated the buttons on the Jetpack page (when not yet connected), and added hover styles to all green buttons.

Screenshots of those changes [here](https://cloudup.com/cAwQoSe9YZM).

Don't worry about the color variable names—I'll be standardizing them to be aligned with the Calypso color naming conventions in a separate PR (they will be $green-jetpack, $green-medium, and $green-dark). I've already defined and used $green-dark in my latest commit though, since that's the new hover/focus button border color.

cc @MichaelArestad @johnHackworth for review.

File: _inc/client/components/connect-button/style.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+@import "../../scss/variables/_colors";
+
+// ==========================================================================
+// Buttons
+// ==========================================================================
+
+.jp-jetpack-connect__button {
+	background: $green-primary;
+    border-color: $green-secondary;
+    color: $white;
```
Comment (zinigor) [inline _inc/client/components/connect-button/style.scss]: Indentation seems to be using different styles here. Let's use tabs please.

**Comment by zinigor** [comment]
Now I'm seeing buttons with new colors in different UI, are these buttons supposed to be like this? One is green, one is blue on the disconnected dashboard view:
screenshot from 2018-01-22 14-30-49

**Comment by MichaelArestad** [comment]
> are these buttons supposed to be like this?

@zinigor This is intentional.

**Comment by johnHackworth** [comment]
@zinigor thank you for the review, Igor!

I fixed the spaces issue in the scss file

File: _inc/client/components/connect-button/style.scss (hunk 1 above)
Comment (joanrho) [reply _inc/client/components/connect-button/style.scss]: That is my mistake. Thanks for catching!

**Pull request merged**